client/src/components/HistoryViewer/HistoryViewerVersionState.js

```diff
@@ -18,16 +18,27 @@ class HistoryViewerVersionState extends Component {
   }
 
   /**
-   * When the record is published, return "Published", else return "Saved"
+   * Return the type of action that was performed when the record was saved
    *
-   * @returns {string}
+   * @returns {string} Returns either Saved, Created, Archived, Unpublished or Published
    */
-  getPublishedState() {
+  getVersionStage() {
     const { version } = this.props;
 
+    if (version.Version === 1) {
+      return i18n._t('HistoryViewer.Created', 'Created');
+    }
+
     if (version.Published) {
+      if (version.Deleted) {
+        if (version.Draft) {
+          return i18n._t('HistoryViewer.Archived', 'Archived');
+        }
+        return i18n._t('HistoryViewer.Unpublished', 'Unpublished');
+      }
       return i18n._t('HistoryViewer.Published', 'Published');
     }
```

Comment:
Do you need to nest these three if statements? IMO it would be cleaner if you didn't

Reply:
I tried the other way and personally I found this way to make more sense.

Reply:
Ok cool, I guess if you read the logic verbatim then not nesting it wouldn't be that pretty

Reply:
I agree with Robbie here, and not nesting would be consistent with the line above where "Created" is returned.